wiki/People/Users_with_unique_titles/en.md

```diff
@@ -274,6 +274,7 @@ These players were awarded their respective titles through either a crazy gamepl
 | ::{ flag=NL }:: [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | Awarded by ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) in late 2008 for reaching rank #104 in 1 week, as a reference to [*The World Ends with You*](https://en.wikipedia.org/wiki/The_World_Ends_with_You) <!-- source: https://osu.ppy.sh/community/forums/posts/60664 (Shirotora_old's forum signature) --> |
 | ::{ flag=US }:: [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Played all Ranked osu! maps as of 27 April 2019 |
 | ::{ flag=DE }:: [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter | Won [The Hunt for the Girl in the Forest](https://osu.ppy.sh/community/forums/topics/662279) |
+| ::{ flag=FI }:: [Tactic](https://osu.ppy.sh/users/6291386) | Prince of disco | Achieved an unbeatable #1 score on [DISCO★PRINCE](https://osu.ppy.sh/beatmapsets/1) |
```

Comment:
where are you getting the star from? it's not in the unicode title

```suggestion
| ::{ flag=FI }:: [Tactic](https://osu.ppy.sh/users/6291386) | Prince of disco | Achieved the maximum possible score on [Kenji Ninuma - DISCO PRINCE](https://osu.ppy.sh/beatmapsets/1), the first Ranked beatmap in the game, gaining them an unbeatable #1 leaderboard spot. |
```

Reply:
it's broken on osu! web, in-game and in the .osu it displays like this: 
![image](https://github.com/ppy/osu-wiki/assets/94870563/f010770f-b696-43d6-b6dc-92b6d6ea2549)

Reply:
star should be there

Reply:
the wiki uses romanised everywhere anyway

Reply:
I've updated it to the wording that you've used, but left the star in because it looks like it's more of an osu-web issue that it's missing :P